src/Tribe/Tickets_Handler.php

```diff
@@ -1195,6 +1195,18 @@ public function get_event_rsvp_tickets( $post = null ) {
 	 *             (allows zero but not `-1` for Unlimited). If oversold, will be corrected to zero.
 	 */
 	public function get_ticket_max_purchase( $ticket_id ) {
+		/* @var Tribe__Cache $cache */
```

Comment:
:warning: This comment is 50% valid code; is this commented out code?

<sub>hash: 00900324f8a837508b885d6c6df67c2d</sub>